tests/aws/test_network_configuration.py

```diff
@@ -28,23 +28,30 @@ class TestOpenSearch:
 
     @markers.aws.only_localstack
     def test_default_strategy(
-        self, opensearch_wait_for_cluster, assert_host_customisation, aws_client
+        self, opensearch_wait_for_cluster, assert_host_customisation, aws_client, cleanups
```

Comment:
There's a fixture for this that combines `create_domain` and `opensearch_wait_for_cluster`: `opensearch_create_domain`.

Reply:
You are right, I somehow thought you wanted to do the call yourself for a reason, I will replace it with the next commit